packages/gestalt/src/Callout.js

```diff
@@ -164,27 +166,19 @@ export default function Callout({
               marginBottom="auto"
               marginTop="auto"
             >
-              {/* We repeat this code block to ensure that text is 
-              centered for our smaller displays and left aligned 
-              for larger displays */}
-              <Box smDisplay="none">
-                {title && (
-                  <Box marginBottom={2}>
-                    <Heading align="center" size="sm">
-                      {title}
-                    </Heading>
-                  </Box>
-                )}
-                <Text align="center">{message}</Text>
-              </Box>
-              <Box smDisplay="block" display="none">
-                {title && (
-                  <Box marginBottom={2}>
-                    <Heading size="sm">{title}</Heading>
-                  </Box>
-                )}
-                <Text>{message}</Text>
-              </Box>
+              {title && (
+                <Box marginBottom={2}>
+                  <Heading
+                    align={responsiveMinWidth === 'xs' ? 'center' : undefined}
+                    size="sm"
+                  >
+                    {title}
+                  </Heading>
+                </Box>
+              )}
+              <Text align={responsiveMinWidth === 'xs' ? 'center' : undefined}>
```

Comment:
Should we also be doing something like this below for the secondary button where we also use display="none"?

```
          {secondaryLink && responsiveMinWidth !== 'xs' && (
             <CalloutLink type="secondary" data={secondaryLink} />
          )}
          {primaryLink && <CalloutLink type="primary" data={primaryLink} />}
          {secondaryLink && responsiveMinWidth === 'xs' && (
              <CalloutLink
                type="secondary"
                data={secondaryLink}
                stacked={!!secondaryLink}
              />
          )}
```

Reply:
Good point. DONE